spring-ai-docs/src/main/antora/modules/ROOT/pages/upgrade-notes.adoc

```diff
@@ -5,6 +5,18 @@
 
 * The configuration prefix for the Chroma Vector Store has been changes from `spring.ai.vectorstore.chroma.store` to `spring.ai.vectorstore.chroma` in order to align with the naming conventions of other vector stores.
 
+* The default value of the `initialize-schema` property on vector stores capable of initializing a schema is now set to `false`.
+This implies that the applications now need to explicitly opt-in for schema initialization on supported vector stores, if the schema is expected to be created at application startup.
+Not all vector stores support this property.
+See the corresponding vector store documentation for more details.
+The following are the vector stores that currently don't support the `initialize-schema` property.
+
+1. Cassandra
```

Comment:
Cassandra does support this property.

`CassandraVectorStoreProperties extends CommonVectorStoreProperties` and the bean definition for the `CassandraVectorStore` in `CassandraVectorStoreAutoConfiguration` has the following code
```
		if (!properties.isInitializeSchema()) {
			builder = builder.disallowSchemaChanges();
		}
```

So the